src/features/vault/components/Pools/Pools.js

```diff
@@ -77,6 +92,14 @@ export default function Pools() {
               <TVLLoader className={classes.titleLoader} />
             )}
           </span>
+
+          {fetchBifibuybackDone && chainBifibuyback && (
+            <span className={classes.text}>
+              {t('Vault-Bifibuyback')} {formatGlobalTvl(chainBifibuyback)}
+              {'/'}
+              {t('Vault-BifibuybackDaySuffix')}
```

Comment:
We can make this simpler to translate if we can change this to
```js
{t('Vault-BifiBuyback', {amount: formatGlobalTvl(chainBifibuyback)})}
````

And the translation to
```json
"Vault-BifiBuyback": "BIFI Buyback {{amount}}/day",
```

Reply:
Good idea